lib/modules/manager/composer/readme.md

```diff
@@ -1,3 +1,10 @@
+<!-- prettier-ignore -->
+!!! warning
+    Renovate does not support Composer version `2.5.0` or newer.
+    This is because of changes in the default behavior of Composer.
```

Comment:
Please check my wording on this, we want to avoid blame, but also be honest about what's breaking for us? Maybe we can explain how the change breaks our workflow/code?